ParquetRoller/Roller.cs

```diff
@@ -349,24 +357,40 @@ private static ExitCode DisplayBadArguments(ModelCollection inWorkload)
         /// Lists the defined ranges for the given <see cref="Model"/>s' <see cref="ModelID"/>s.
         /// </summary>
         /// <param name="inWorkload">The <see cref="Model"/>s to inspect.</param>
-        /// <returns><see cref="ExitCode.BadArguments"/></returns>
+        /// <returns><see cref="ExitCode.Success"/></returns>
         private static ExitCode ListRanges(ModelCollection inWorkload)
         {
-            // TODO This is a stub.
-            Console.WriteLine(inWorkload);
-            return ExitCode.BadArguments;
+            if (inWorkload == null || inWorkload.Count == 0)
+            {
+                Console.WriteLine("No defined content.");
+                return ExitCode.Success;
+            }
+
+            var orderedWorkload = inWorkload.OrderBy(x => x.ID);
+
+            Console.WriteLine($"Minimum: {orderedWorkload.First().ID}");
```

Comment:
This isn't exactly what I wanted this method to do.  
What I was trying to convey here is a listing of the IDs that _could_ be used (i.e., the bounds of the collection), rather than the IDs that are in use (i.e., the IDs actually in the collection).

However, in looking at it I realized several things that I had coded previously prevented the intended behavior, including visibility of a getter in the class library.  So, I went ahead and rewrote it myself.

I'll submit that as a followup commit.